src/modules/direction/store/directionSlice.ts

```diff
@@ -291,7 +291,7 @@ export const fetchMaterials = (direction: IDirection): AppThunkType => async (
 
     const response = await getPosts('latest-by-direction', {
       params: {
-        direction: direction.id,
+        direction: +direction.id,
```

Comment:
this one is already a number, see IDirection
```suggestion
        direction: direction.id,
```

Reply:
fixed